Nos/Views/FullscreenProgressView.swift

```diff
@@ -6,18 +6,23 @@
 //
 
 import SwiftUI
+import SwiftUINavigation
 
 struct FullscreenProgressView: View {
     
     @Binding var isPresented: Bool 
-    
+
+    var text: TextState?
     var hideAfter: DispatchTime?
     
     var body: some View {
         VStack {
             Spacer()
             ProgressView()
                 .foregroundColor(.primaryTxt)
+            if let text {
+                Text(text)
```

Comment:
The progress wheel and text look a bit too close for my eye. Could you add some padding to the text?
```suggestion
                Text(text).padding()
```

Reply:
[497e629](https://github.com/planetary-social/nos/pull/492/commits/497e629f9c55f85774cf9b84da2ff069ff80cd3a)